tests/study_tests/test_study.py

```diff
@@ -661,15 +662,15 @@ def objective(trial: Trial) -> float:
         with pytest.warns(UserWarning):
             study.optimize(objective, n_trials=1)
         t = study.trials[0]
-        assert -10 <= t.params["x"] <= 10
+        assert t.params["x"] == fixed_value
```

Comment:
After this PR, enqueued parameter is not ignored even when it is out of range from the search space.